TechTalk.SpecFlow/Infrastructure/TestExecutionEngine.cs

```
@@ -147,8 +152,16 @@ public void OnAfterLastStep()
                 _testTracer.TraceDuration(duration, "Scenario: " + _contextManager.ScenarioContext.ScenarioInfo.Title);
             }
 
+            var testResultResult = _testResultFactory.BuildFromScenarioContext(_contextManager.ScenarioContext);
+            if (testResultResult is ISuccess<TestResult> success)
```

we swallow the failure.
We should at least write it to the testTracer